tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go

```diff
@@ -102,15 +66,58 @@ var _ = Describe(
 				_, err = newIbguBuilder.WaitUntilComplete(30 * time.Minute)
 				Expect(err).NotTo(HaveOccurred(), "Prep and Upgrade IBGU did not complete in time.")
 
+				upgradeSuccessful = true
 			})
 
 			By("Saving target sno cluster info after upgrade", func() {
 				err := cnfclusterinfo.PostUpgradeClusterInfo.SaveClusterInfo()
 				Expect(err).ToNot(HaveOccurred(), "Failed to collect and save target sno cluster info after upgrade")
 			})
 
+			By("Running post-upgrade validations", func() {
+				cnfibuvalidations.PostUpgradeValidations()
+			})
 		})
 
-		cnfibuvalidations.PostUpgradeValidations()
+		if upgradeSuccessful {
```

Comment:
if condition should be added inside the rollback test function, with proper skip message. So it will not be skipped silently.

Reply:
Fixed- thank you.